wrapper/Mathcad/README.md

```
@@ -54,39 +54,51 @@ Make the Build for Mathcad Prime (any version above 3.0)
     cmake .. -DIF97_MATHCAD15_MODULE=ON 
              -DIF97_MATHCAD15_ROOT="C:/Program Files (x86)/Mathcad/Mathcad 15"  
              -G "Visual Studio 17 2022" -A Win32 
              -DCMAKE_VERBOSE_MAKEFILE=ON 
-	     
-	> Insert your version of Visual Studio for the -G option.  
-    > Legacy Mathcad was 32-bit and requires a 32-bit add-in DLL.  
-    > Prior to VS2017, use something like `-G "Visual Studio 14 2015` as 32-bit was the default.
+	```
+         
+	- Insert your version of Visual Studio for the -G option.  
+    - Legacy Mathcad was 32-bit and requires a 32-bit add-in DLL.  
```

markdownlint-cli2 (0.22.1)

[warning] Inconsistent indentation for list items at the same level
Expected: 1; Actual: 4

(MD005, list-indent)